packages/framesync/src/types.ts

```diff
@@ -15,13 +15,7 @@ export interface Step {
   process: (frame: FrameData) => void;
 }
 
-export enum StepId {
-  Read = 'read',
-  Update = 'update',
-  Render = 'render',
-  PostRender = 'postRender',
-  FixedUpdate = 'fixedUpdate'
```

Comment:
I've removed `fixedUpdate` entirely - it seems unused, couldnt find it anywhere.